geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/ConverterIngest.scala

```diff
@@ -104,8 +98,11 @@ class ConverterIngest(dsParams: Map[String, String],
               toWrite.getUserData.put(Hints.USE_PROVIDED_FID, java.lang.Boolean.TRUE)
               try {
                 fw.write()
+                written.incrementAndGet()
```

Comment:
you can call ```AccumuloFeatureWriter.flush()``` and it will pause until the mutations are sent to the tablet servers (and else throw a MutationsRejectedException) (as far as I remember)